seleniumbase/core/browser_launcher.py

```diff
@@ -1731,6 +1746,7 @@ def get_local_driver(
                 headless,
                 locale_code,
                 proxy_string,
+                proxy_bypass_list,
                 proxy_auth,
                 proxy_user,
                 proxy_pass,
```

Comment:
Looks like ``proxy_bypass_list`` is not in the correct location here. Should be after ``proxy_pass``.